Dockerfile

```diff
@@ -183,6 +183,19 @@ RUN \
     && apk del .build-dependencies \
     && rm -rf /usr/src/telldus
 
+
+# iperf3

```

Comment:
Can you add also a link to the issue? So that we knew in which version we can move back to upstream.

Reply:
There is no issue, added a reference to the PR in <https://github.com/home-assistant/docker/pull/203>